### ISSUE TITLE
ver 0.3.7-AndroidRuntimeException: Calling startActivity() from outside of an Activity...

### ISSUE THREAD

**Issue by devsideal** [FolioReader/FolioReader-Android#200]
I'm using library version 0.3.7
When trying to open epub from my Activity, eg -
```
FolioReader folioReader = FolioReader.getInstance(getApplicationContext());
folioReader.openBook(R.raw.barrett); 
```
Getting this error -
(While i'm opening the file from my activity)
```
android.util.AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?
        at android.app.ContextImpl.startActivity(ContextImpl.java:672)
        at android.app.ContextImpl.startActivity(ContextImpl.java:659)
        at android.content.ContextWrapper.startActivity(ContextWrapper.java:331)
        at com.folioreader.FolioReader.openBook(FolioReader.java:97)
        at com.devs.ncert.activity.MainActivity$1.onClick(MainActivity.java:35)
............
```

**Comment by mahavir155** [comment]
@devsideal This issue is fixed in latest version 0.3.8..

**Comment by devsideal** [comment]
Yes this is fixed in 0.3.7 , now app is crashing with this exception

**Comment by KishanViramgama** [comment]
0.3.9 same issue
